run-queue

```diff
@@ -60,12 +60,12 @@ def consume_webhook_queue(channel, amqp):
             cmd = ['./issue-scan', '--issues-data', json.dumps(request), '--amqp', amqp]
         elif action in ['opened', 'synchronize']:
             cmd = ['./tests-scan', '--pull-data', json.dumps(request), '--amqp', amqp, '--repo', repo]
-        # When PR was merged, generate task for storing failures
+        # When PR was merged, generate task for storing tests
         elif action == 'closed' and pull_request.get('merged', False):
             sha = pull_request['head']['sha']
-            db = os.path.join(get_images_data_dir(), "test-failures.db")
+            db = os.path.join(get_images_data_dir(), "test-results.db")
             data = os.path.join(get_images_data_dir(), "failure-stats.js")
-            cmd = "./store-failures --db {0} --repo {1} {2} && ./generate-stats --db {0} > {3}".format(
+            cmd = "./store-tests --db {0} --repo {1} {2} && ./generate-stats --db {0} > {3}".format(
```

Comment:
Where will the html actually be copied? I expect that we'll still change this quite a bit, so having to copy it manually is a bit cumbersome.

Reply:
For now it is manually copied. Once that becomes burden, we can surely do it somewhere in bots.